server/config/routes.js

```diff
@@ -36,6 +39,10 @@ module.exports.routes = {
   "post /api/auth/login": "auth/post-login",
   "post /api/auth/signup": "auth/post-signup",
   "post /api/auth/signup-google": "auth/post-signup-google",
+  
```

Comment:
can you put a comment like the others just to help fix merge conflicts in the future